general.php

```diff
@@ -46,6 +46,8 @@
 
         'useCompressedJs' => (getenv('CRAFT_USE_COMPRESSED_JS') ?: 'true') === 'true',
 
-        'userSessionDuration' => (int) (getenv('CRAFT_USER_SESSION_DURATION') ?: '3600')
+        'userSessionDuration' => (int)(getenv('CRAFT_USER_SESSION_DURATION') ?: '3600'),
+
+        'sendPoweredByHeader' => (getenv('CRAFT_SEND_POWERED_BY_HEADER') ?: 'true') === 'true'
```

Comment:
I think this is the only line that actually contributes to the fix, right?

Is it possible your IDE has made some extra changes automatically? In order to slimline the diff and allow better history, are we able to just add this one line?

Reply:
No I made the other changes too as the formatting was incorrect, I can do this in a separate commit if you would prefer